pkg/config/config.go

```diff
@@ -1017,6 +1020,8 @@ func InitConfig(config Config) {
 	config.BindEnv("orchestrator_explorer.max_message_bytes")
 	config.BindEnv("orchestrator_explorer.orchestrator_dd_url")
 	config.BindEnv("orchestrator_explorer.orchestrator_additional_endpoints")
+	// add additional env var name
+	config.BindEnv("orchestrator_explorer.orchestrator_additional_endpoints", "DD_ORCHESTRATOR_ADDITIONAL_ENDPOINTS")
```

Comment:
This is a duplicate from the above line

Reply:
Intentional duplicate to also bind the `DD_ORCHESTRATOR_ADDITIONAL_ENDPOINTS` name in addition to the default.

Reply:
indeed 👍